aws/policy/paas.yaml

```diff
@@ -94,6 +94,10 @@ Statement:
       - lightsail:RebootInstance
       - lightsail:StopInstance
       - lightsail:ReleaseStaticIp
+      - lightsail:PutInstancePublicPorts
+      - lightsail:GetInstanceSnapshot
+      - lightsail:CreateInstanceSnapshot
```

Comment:
Could you move the `lightsail:CreateInstanceSnapshot` up to the `AllowResourceRestrictedActionsWhichIncurFees` Sid? And can you also add `lightsail:GetInstanceSnapshots` here? Otherwise, looks good.

Reply:
I did not notice the other Sid. Its done!